### PR TITLE
rename tendrl-frontend to dashboard

### PR DESCRIPTION
**Pull request by kamleshverma1**

### CODE REVIEW

**Comment by kamleshverma1** [comment]
@gnehapk @cloudbehl Please review it

**Comment by kamleshverma1** [comment]
linked to https://github.com/Tendrl/dashboard/issues/62

**Comment by gnehapk** [comment]
Looks good to me. @kamleshverma1  Please test rpm with these changes.

**Comment by gnehapk** [comment]
@TimothyAsir Please review.

**Comment by TimothyAsir** [comment]
Looks good to me, Please build the rpm once with the fresh clone with this patch and check you are able to build the rpm successfully with new name.

**Comment by gnehapk** [comment]
@kamleshverma1 please verify it with fresh clone as per @TimothyAsir comment.

**Comment by TimothyAsir** [comment]
also please rename the spec file name to tendrl-dashboard.spec or dashboard.spec whichever is convenient for you.

File: tendrl-frontend.spec [hunk 1]
```diff
@@ -1,12 +1,12 @@
-Name: tendrl-frontend
+Name: dashboard
```
Comment (r0h4n) [inline tendrl-frontend.spec]: - rpm names require product prefix. Make this "tendrl-dashboard"
- Rename this file to "tendrl-dashboard.spec"

**Comment by kamleshverma1** [comment]
@r0h4n @TimothyAsir incorporated  your comments ..Please review it again .
tested with manually building rpm with fresh clone . worked well! only tendrl-api call getting break due to not latest tendrl-api rpm and redirection.

**Comment by TimothyAsir** [comment]
looks good to me

**Pull request merged**